packages/schematics/angular/migrations/update-8/index.ts

```diff
@@ -0,0 +1,85 @@
+/**
```

Comment:
Probably it makes sense that in this rule we update the codelyzer package aswell.

Reply:
👍 done